staging/src/k8s.io/apiserver/pkg/registry/rest/meta.go

```diff
@@ -40,7 +40,7 @@ func FillObjectMetaSystemFields(meta metav1.Object) {
 		if err != nil {
 			uid := uuid.NewUUID()
 			meta.SetUID(uid)
-			klog.Infof("Got invalid uuid [%s]. Assign a new one [%v]", uidStr, uid)
+			klog.V(3).Infof("Got invalid uuid [%s]. Assign a new one [%v]", uidStr, uid)
```

Comment:
just to follow up, what is fix to avoid this invalid UUID error?

Reply:
Actually, we have not found the solution yet. 

I tried the following two, none of them working
1. let each static pod create a real uid. Then the hollow nodes fail to get ready.
2.  change the format of static format to the expected. The density test failed as some static pods are not in the right states.